components/CurrentlyReadingAside.tsx

```diff
@@ -0,0 +1,24 @@
+import { BookItems } from "@/config/BookItems";
+import BookCurrentElement from "./BookCurrentElement";
+
+function CurrentlyReadingAside() {
+  return (
+    <aside className="flex flex-col w-1/4  items-start h-screen p-4 overflow-hidden">
```

Comment:
This is awesome that you use different elements than `div`s, very good and handy for accessibility 👏